### PR TITLE
modules/utils normalizePath: fixed test for windows env

### LINKED ISSUE #1041: Fix AppVeyor relative path fail
```
 1) modules/utils normalizePath returns the relative path resolved against the base path:
     AssertionError: false == true
      at Context.<anonymous> (C:\projects\node-jscs\test\utils.js:150:20)
      at callFn (C:\projects\node-jscs\node_modules\mocha\lib\runnable.js:251:21)
      at Test.Runnable.run (C:\projects\node-jscs\node_modules\mocha\lib\runnable.js:244:7)
      at Runner.runTest (C:\projects\node-jscs\node_modules\mocha\lib\runner.js:374:10)
      at C:\projects\node-jscs\node_modules\mocha\lib\runner.js:452:12
      at next (C:\projects\node-jscs\node_modules\mocha\lib\runner.js:299:14)
      at C:\projects\node-jscs\node_modules\mocha\lib\runner.js:309:7
      at next (C:\projects\node-jscs\node_modules\mocha\lib\runner.js:248:23)
      at Immediate._onImmediate (C:\projects\node-jscs\node_modules\mocha\lib\runner.js:276:5)
      at processImmediate [as _immediateCallback] (timers.js:321:17)
```

cc @mrjoelkemp @mdevils see: https://ci.appveyor.com/project/mdevils/node-jscs/build/2

<bountysource-plugin>

---
Want to back this issue? **[Place a bounty on it!](https://www.bountysource.com/issues/8465956-fix-appveyor-relative-path-fail?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by zxqfox**
Fixes #1041

### CODE REVIEW

File: test/utils.js [hunk 1]
```diff
@@ -147,7 +147,7 @@ describe('modules/utils', function() {
         });
 
         it('returns the relative path resolved against the base path', function() {
-            assert.ok(utils.normalizePath('../foo', base) === (path.dirname(base) + '/foo'));
+            assert.equal(utils.normalizePath('../foo', base), path.resolve(path.dirname(base) + '/foo'));
```
Comment (mrjoelkemp) [inline test/utils.js]: Maybe `path.resolve(path.dirname(base), 'foo'))`?

Comment (zxqfox) [inline test/utils.js]: Thanks ;-) fixed.

**Comment by mrjoelkemp** [comment]
If this actually fixes things on Windows, then :+1:

**Comment by zxqfox** [comment]
@mrjoelkemp Yep, it is ;-) As you can see both builds succeed.

**Comment by mrjoelkemp** [comment]
Oh wow, didn't notice the "2 successful checks." 

LGTM

**Pull request closed without merging**